### PR TITLE
Updated hamburger menu

### PR DESCRIPTION
**Pull request by ranisa-gupta16**
## JIRA Ticket

https://jira.csiro.au/browse/BSS-265
Subtasks : 1 - **https://jira.csiro.au/browse/BSS-295**
                 2 - **https://jira.csiro.au/browse/BSS-296**
                 3 - **https://jira.csiro.au/browse/BSS-297**

## Description

Updated the hamburger menu/side navigation menu to align with our new design specifications. This update presents an opportunity for us to review the menu structure and discuss which options should be included or excluded.

## Proposed Changes

 - New Menu items added to the existing hamburger/side-nav menu
 - Menu items added :
     - Help
     - Switch Org (Have a placeholder based on a check whether you're part of different org)
     - Sign out being directed to where user was directed to. 
- Enhanced styling, added border, shadow to make it look distinct from content page
- Laid out top and bottom menu items as according to Figma design
- Added placeholder routes for switch org and help buttons which can be simply used in future.

## How to Test
Just run the app and you should be able to see the new navigation menu to the left.
If you click on help, switch org, sign-out, you should see a blank meaning the respective components will be connected to them.

## Additional Information

Currently, there is a profile option as per the Figma design. There's some ambiguity regarding what should be included under the profile section so that option hasn't been included in this design.

Notification is not in scope for V1.

For now, I've linked the sign-out option that redirects to the user page where you can sign out which was previously displayed as User.

## Checklist

- [ Y]  I have confirmed all commits have been signed.
- [ Y ] I have added JSDoc style comments to any new functions or classes.
- [ Y ] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

File: app/src/gui/layout/appBar.tsx [hunk 1]
```diff
@@ -197,45 +251,64 @@ export default function MainAppBar(props: NavbarProps) {
...
+
+    // @TODO Ranisa: Commented this for now, to be discussed if needed to display this nav item
+    // {
+    //   title: 'About Build',
+    //   icon: <SettingsIcon />,
+    //   to: ROUTES.ABOUT_BUILD,
+    //   disabled: false,
+    // },
+
     {
```
Comment (stevecassidy) [inline app/src/gui/layout/appBar.tsx:279]: We definitely need to keep this menu item, important for app management, dumping data and bug reporting.

File: app/src/gui/layout/appBar.tsx [hunk 2]
```diff
@@ -19,8 +19,8 @@
  *   throughout the app.
  */
 
-import React, {useEffect, useState} from 'react';
-import {Link as RouterLink, NavLink} from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { Link as RouterLink, NavLink } from 'react-router-dom';
 import {
```
Comment (stevecassidy) [inline app/src/gui/layout/appBar.tsx:24]: You'll need to merge in the most recent changes and run lint to get everything aligned with the gts code style.

File: app/src/gui/layout/appBar.tsx [hunk 3]
```diff
@@ -197,45 +251,64 @@ export default function MainAppBar(props: NavbarProps) {
...
+      to: ROUTES.HELP,
       disabled: false,
     },
+
+    {
+      title: 'Switch org',
+      icon: <SwapHorizIcon />,
+      to: ROUTES.SWITCH_ORG,
+      disabled: false,
+    },
```
Comment (stevecassidy) [inline app/src/gui/layout/appBar.tsx:291]: I don't think we should add this until we have this functionality. Confusing for users if it does nothing.

**Review by stevecassidy — CHANGES_REQUESTED** [review]
I think some of these changes are premature. We need to maintain the main branch as a releasable app for users. New menu items should only go in once the functionality they support is present.

File: app/src/gui/layout/appBar.tsx (hunk 3 above)
Comment (ranisa-gupta16) [reply app/src/gui/layout/appBar.tsx:286]: Got your point. Removing this and help.

File: app/src/gui/layout/appBar.tsx (hunk 1 above)
Comment (ranisa-gupta16) [reply app/src/gui/layout/appBar.tsx:273]: Noted

**Review by stevecassidy — APPROVED** [review]
Looks good.

**Pull request merged**